apps/studio/src/components/connection/PostgresForm.vue

```diff
@@ -16,7 +13,54 @@
         type="text"
         class="form-control"
         v-model="config.options.cluster"
+      />
+    </div>
+    <common-advanced :config="config" />
+
+    <div class="advanced-connection-settings">
```

Comment:
is this only for Redshift? Does it work for RDS Postgres also, or is that out of scope of this PR?

Reply:
I tested on RDS Postgres